Sources/ImagePipeline.swift

```diff
@@ -278,6 +289,21 @@ private extension ImagePipeline {
         }
     }
 
+    func didReceiveEvent(_ event: Task<ImageResponse, Error>.Event, task: ImageTask) {
+        switch event {
+        case let .error(error):
```

Comment:
This could be implemented as a pure mapping from `Task<ImageResponse, Error>.Event` to `ImageTaskEvent`.